nav2_costmap_2d/src/clear_costmap_service.cpp

```diff
@@ -36,21 +37,35 @@ ClearCostmapService::ClearCostmapService(rclcpp::Node::SharedPtr & node, Costmap
 
   node_->get_parameter_or_set("clearable_layers", clearable_layers_, {"obstacle_layer"});
 
-  server_ = node_->create_service<ClearExceptRegion>("clear_except_" + costmap_.getName(),
+  except_region_server_ = node_->create_service<ClearExceptRegion>("clear_except_" + costmap_.getName(),
```

Comment:
I'd make these names match the service name. eg. `clear_except_region_service_` or just `clear_except_region_` and `clear_entire_service`

Reply:
Good point, will push a change.